03_method/03_try_over3_3.rb

```diff
@@ -37,38 +72,63 @@ def self.included(mod)
           self.class.define_method "#{attr_sym}?" do
             @attr == true
           end
+        elsif respond_to?(:hoge?)
+          self.class.remove_method :hoge?
```

Comment:
```suggestion
        elsif respond_to? "#{atter_sym}?"
          self.class.remove_method "#{atter_sym}?"
```
`hoge?`ではなく、`#{name}?`を削除する必要がある。（汎用的に）